js/MVP/views/landing/Landing.js

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import AlertStripe from 'nav-frontend-alertstriper';
+import styled from 'styled-components';
+import AppSpinner from '../../../components/AppSpinner';
+import { BRUKER } from '../../../enums/moteplanleggerDeltakerTyper';
+import {
+  AVBRUTT,
+  BEKREFTET,
+  erMotePassert,
+  getSvarsideModus,
+  konverterTid,
+  MOTESTATUS,
+} from '../../../utils/moteUtils';
+import DialogmoteContainer from '../../containers/DialogmoteContainer';
+import { brevTypes } from '../../globals/constants';
+import { useBrev } from '../../hooks/brev';
+import { useMotebehov } from '../../hooks/motebehov';
+import { useMoteplanlegger } from '../../hooks/moteplanlegger';
+import { getLongDateFormat } from '../../utils';
+import DialogmoteVideoPanel from './components/DialogmoteVideoPanel';
+import MotebehovPanel from './components/MotebehovPanel';
+import MoteinnkallelsePanel from './components/MoteinnkallelsePanel';
+import MoteplanleggerKvitteringPanel from './components/MoteplanleggerKvitteringPanel';
+import MoteplanleggerPanel from './components/MoteplanleggerPanel';
+import MotereferatPanel from './components/MotereferatPanel';
+import PreviousMotereferatPanel from './components/PreviousMotereferatPanel';
+import VeilederLanding from './components/VeilederLanding';
+
+const AlertStripeStyled = styled(AlertStripe)`
+  margin-bottom: 32px;
+`;
+
+const Landing = () => {
+  const brev = useBrev();
+  const motebehov = useMotebehov();
+  const moteplanlegger = useMoteplanlegger();
+
+  if (brev.isLoading || motebehov.isLoading || moteplanlegger.isLoading) {
+    return <AppSpinner />;
+  }
+
+  const FetchFailedError = () => {
+    if (brev.isError || motebehov.isError || (moteplanlegger.isError && !(moteplanlegger.error.message === '404'))) {
+      return (
+        <AlertStripeStyled type="feil">
+          Akkurat nå mangler det noe her. Vi har tekniske problemer som vi jobber med å løse. Prøv gjerne igjen om en
+          stund.
+        </AlertStripeStyled>
+      );
+    }
+
+    return null;
+  };
+
+  const harSammeAvlysningsstatus = (brevType, moteplanleggerStatus) => {
+    return (
+      (brevType === brevTypes.AVLYST && moteplanleggerStatus === AVBRUTT) ||
+      (brevType !== brevTypes.AVLYST && moteplanleggerStatus !== AVBRUTT)
+    );
+  };
+
+  const displayBrev = () => {
+    if (brev.isError || brev.data.length === 0) {
+      return false;
+    }
+
+    if (!moteplanlegger.isError && moteplanlegger.data) {
+      const brevDatoArraySorted = brev.data.map((i) => new Date(i.createdAt)).sort((a, b) => b - a);
+      const sistOpprettetBrev = brevDatoArraySorted[0];
+      const sistOpprettetMoteplanleggerMoteTidspunkt = new Date(moteplanlegger.data.opprettetTidspunkt);
+
+      if (harSammeAvlysningsstatus(sistOpprettetBrev.brevType, moteplanlegger.data.status)) {
+        return sistOpprettetBrev > sistOpprettetMoteplanleggerMoteTidspunkt;
+      }
+    }
+    return true;
+  };
+
+  const displayMotebehov = () => {
```

Comment:
Litt tung å lese pga negeringer og mange if-sjekker. Mulig å forenkle denne noe?

Reply:
Enig, vi tørr ikke helt å røre den nå, som er et stort symptom på problemet du nevner :P

Etter deploy? Kan være greit å ta det når vi skriver tester

Reply:
Veldig for å forsikre oss med tester